.editorconfig

```diff
@@ -0,0 +1,88 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+# Change these settings to your own preference
+indent_style = space
+indent_size = 4
+
+# We recommend you to keep these unchanged
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.feature]
```

Comment:
Why not gathering the files with the same style?

Reply:
to make it more understandable

Reply:
Why not, it's subjective anyway :slightly_smiling_face: